### PR TITLE
Updates readme for new deployment flow

### LINKED ISSUE #261: Update README about new deployment process

### PR DESCRIPTION
**Pull request by Frijol**
resolves https://github.com/edgi-govdata-archiving/EEW-Website/issues/261

@lightandluck is this all correct? Anything I forgot to update?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,9 +3,9 @@
 
 # EEW Website
 
-A website for the [Environmental Enforcement Watch](//environmentalenforcementwatch.org) project by [EDGI](//envirodatagov.org)
+The environmentalenforcementwatch.org website for the [Environmental Enforcement Watch](//environmentalenforcementwatch.org) project by [EDGI](//envirodatagov.org)
 
-Currently staged at https://edgi-govdata-archiving.github.io/EEW-Website/
+Staging site at [staging.environmentalenforcementwatch.org](staging.environmentalenforcementwatch.org)
```
Comment (lightandluck) [inline README.md]: Link needs `//` like above links. Currently tries to link to a github page with that path and 404s.

File: README.md [hunk 2]
```diff
@@ -26,23 +26,24 @@ Currently staged at https://edgi-govdata-archiving.github.io/EEW-Website/
...
 ### Toolset
 * Written in [Gatsby](https://www.gatsbyjs.org/), a Node React framework
-* Hosted on Github Pages
+* Hosted on Namecheap
 * Continuous deployment uses Github Actions
 
 ### Deployment
 #### Continuous deployment
-The site currently deploys automatically on each merge to the `main` branch using Github Actions.
+This site deploys automatically to [staging](staging.environmentalenforcementwatch.org) upon merge to `main`, and to [production](environmentalenforcementwatch.org) upon tagged release.
```
Comment (lightandluck) [inline README.md]: Needs `//` for links

File: README.md [hunk 3]
```diff
@@ -26,23 +26,24 @@ Currently staged at https://edgi-govdata-archiving.github.io/EEW-Website/
...
+##### Staging
+Staging automatic deployment configuration can be found in `.github/workflows/continuous-integration-workflow.yml`.
 
-To debug, check the Actions tab of this repo.
+Upon merge to main, the staging Github Action is triggered which builds the site to the gh-pages branch and ftp deploys it to [staging.environmentalenforcementwatch.org](staging.environmentalenforcementwatch.org)
 
-#### Manual deployment
-The site can be manually deployed using the npm module [gh-pages](https://github.com/tschaub/gh-pages) as recommended by Gatsby [here](https://www.gatsbyjs.com/docs/how-gatsby-works-with-github-pages/#deploying-to-a-github-pages-subdomain-at-githubio).
+##### Production
+Staging automatic deployment configuration can be found in `.github/workflows/release-to-production.yml`.
```
Comment (lightandluck) [inline README.md]:
```suggestion
Production automatic deployment configuration can be found in `.github/workflows/release-to-production.yml`.
```

File: README.md [hunk 4]
```diff
@@ -26,23 +26,24 @@ Currently staged at https://edgi-govdata-archiving.github.io/EEW-Website/
...
 
-Ensure you have dev dependencies installed and then run `npm run deploy` from a local clone of the `main` branch.
+Upon a tagged release (through the “Releases” section of the repo), production Github action is triggered which ftp deploys the gh-pages branch to [environmentalenforcementwatch.org](environmentalenforcementwatch.org)
```
Comment (lightandluck) [inline README.md]: Link needs `//`

**Review by lightandluck — CHANGES_REQUESTED** [review]
Just need to update the links so they direct properly. Thank you sooo much for doing this!

File: README.md (hunk 1 above)
Comment (Frijol) [reply README.md]: ha, thanks for picking that up

Comment (Frijol) [inline README.md]:
```suggestion
Staging site at [staging.environmentalenforcementwatch.org](//staging.environmentalenforcementwatch.org)
```

File: README.md (hunk 2 above)
Comment (Frijol) [inline README.md]:
```suggestion
This site deploys automatically to [staging](//staging.environmentalenforcementwatch.org) upon merge to `main`, and to [production](//environmentalenforcementwatch.org) upon tagged release.
```

File: README.md (hunk 4 above)
Comment (Frijol) [inline README.md]:
```suggestion
Upon a tagged release (through the “Releases” section of the repo), production Github action is triggered which ftp deploys the gh-pages branch to [environmentalenforcementwatch.org](//environmentalenforcementwatch.org)
```

**Pull request merged**